build.gradle.kts

```diff
@@ -39,6 +44,22 @@ kotlin {
     explicitApiWarning()
 }
 
+tasks.withType<DokkaTask>().configureEach {
+    moduleName.set("Weave Loader")
+    dokkaSourceSets.configureEach {
+        sourceLink {
+            localDirectory.set(projectDir.resolve("src"))
+            remoteUrl.set(URL("https://github.com/Weave-MC/Weave-Loader/tree/master/src"))
+            remoteLineSuffix.set("#L")
+        }
+        externalDocumentationLink("https://kotlinlang.org/api/kotlinx.serialization/", "https://kotlinlang.org/api/kotlinx.serialization/package-list")
+        externalDocumentationLink("https://asm.ow2.io/javadoc/", "https://asm.ow2.io/javadoc/element-list")
+        externalDocumentationLink("https://jenkins.liteloader.com/job/Mixin/javadoc/","https://jenkins.liteloader.com/job/Mixin/javadoc/package-list")
+        externalDocumentationLink("https://projectlombok.org/api/","https://projectlombok.org/api/element-list")
```

Comment:
Lombok is not exposed as public API, should this be here? Same might be for annotations and serialization